html-forms/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="">
+      <div>
+        Email
+      </div>
+      <div>
+        <label for="email">
+          <input type="text" name="email">
+        </label>
+      </div>
+      <div>
+        <label for="coding-cool">
+          Why is coding cool?
+        </label>
+      </div>
+      <div>
+        <select name="coding">
```

Comment:
The `select` tag should have an `id` with a value that corresponds to the `label`'s `for` attribute.